lib/ui/Popup.tsx

```diff
@@ -40,7 +45,20 @@ function ActivePopup(props: PopupProps & { session: Talk.Session }) {
   useMethod(box, presence, "setPresence");
   useMethod(box, highlightedWords, "setHighlightedWords");
   useConversation(session, box, syncConversation, conversationId, asGuest);
-  useMountBox(box, undefined);
+  const mounted = useMountBox(box, {show: show ?? showOnMount ?? true});
```

Comment:
What do you think about removing `showOnMount` and just having `{show: show ?? true}` instead?

Reply:
Yeah, I agree, it keeps things simpler, cause you can just set `show` to a constant value to achieve the same thing.